build.gradle

```diff
@@ -37,5 +34,27 @@ subprojects { subproject ->
 
     compileJava compileOptions
     compileTestJava compileOptions
+
+    dependencies {
+      compileOnly group: 'io.vertx', name: 'vertx-codegen', version: '3.5.0'
+      compileOnly group: 'io.knotx', name: 'knotx-core', version: '1.3.0-SNAPSHOT'
+
+      testCompile group: 'io.knotx', name: 'knotx-core', version: '1.3.0-SNAPSHOT'
+      testCompile group: 'io.knotx', name: 'knotx-core', version: '1.3.0-SNAPSHOT', classifier: 'tests'
+      testCompile group: 'io.knotx', name: 'knotx-mocks', version: '1.3.0-SNAPSHOT'
+
+      testCompile group: 'io.vertx', name: 'vertx-codegen', version: '3.5.0'
+      testCompile group: 'io.vertx', name: 'vertx-unit', version: '3.5.0'
+      testCompile group: 'junit', name: 'junit', version: '4.12'
+      testCompile group: 'org.powermock', name: 'powermock-api-mockito', version: '1.6.6'
+      testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '1.6.6'
+      testCompile group: 'org.mockito', name: 'mockito-all', version: '2.0.2-beta'
+
+      if (subproject.path.startsWith(":adapter:http")) {
+        compile project(':adapter:api')
+      }
+
```

Comment:
We need also added support for releasing to maven central. see sonatype documenation how to do it